src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -218,8 +237,17 @@ class MAGNUM_BASISIMAGECONVERTER_EXPORT BasisImageConverter: public AbstractImag
 
     private:
         MAGNUM_BASISIMAGECONVERTER_LOCAL ImageConverterFeatures doFeatures() const override;
+
+        MAGNUM_BASISIMAGECONVERTER_LOCAL Containers::Array<char> doConvertToData(Containers::ArrayView<const ImageView1D> imageLevels) override;
         MAGNUM_BASISIMAGECONVERTER_LOCAL Containers::Array<char> doConvertToData(Containers::ArrayView<const ImageView2D> imageLevels) override;
+        MAGNUM_BASISIMAGECONVERTER_LOCAL Containers::Array<char> doConvertToData(Containers::ArrayView<const ImageView3D> imageLevels) override;
+
+        template<UnsignedInt dimensions>
+        bool convertLevelsToFile(const Containers::ArrayView<const BasicImageView<dimensions>> imageLevels, const Containers::StringView filename);
```

Comment:
Minor note, just for consistency: I always put `template<>` on the same line as the signature as I consider it a part of the signature.

Reply:
I fixed it, was an issue across multiple plugins (that I all touched :eyes:) but it didn't come up in previous reviews. Noted for the future.